automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala

```diff
@@ -268,23 +269,37 @@ trait AzureBillingBeforeAndAfter extends FixtureAnyFreeSpecLike with BeforeAndAf
     Some(UUID.fromString("f41c1a97-179b-4a18-9615-5214d79ba600"))
   )
 
+  override def beforeAll(): Unit = {
+    implicit val accessToken = Hermione.authToken().unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
+    val res = for {
+      _ <- IO(super.beforeAll())
+      _ <- withTemporaryAzureBillingProject(azureManagedAppCoordinates, shouldCleanup = false) { projectName =>
+        IO(sys.props.put(azureProjectKey, projectName))
+      }
+//      _ <- IO(sys.props.put(azureProjectKey, "tmp-billing-project-beddf71a74"))
```

Comment:
Could you add this comment above? `// hardcode this if you want to use a static billing profile`